.eslintrc.cjs

```diff
@@ -23,14 +23,27 @@ module.exports = {
     "@babel/development",
     "@babel/development-internal",
   ],
-  extends: "@babel/internal",
+  extends: [
+    "@babel/internal",
+    "plugin:import/recommended",
+    "plugin:import/typescript",
+  ],
   rules: {
     "prettier/prettier": "error",
     "import/no-extraneous-dependencies": "error",
+    "import/no-named-as-default": "off",
+    "import/no-named-as-default-member": "off",
+    "import/no-duplicates": "off",
```

Comment:
It produces a lot of false positives.
most notably
```ts
import type * as N from "../../types";
import type { Expression, Pattern } from "../../types";
```

I'll try to use `typescript-eslint` in a future PR.